types/integra/aquastream/device.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$id": "https://akenza.io/integra/aquastream/device.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Device",
```

Comment:
why is this not called lifecycle as always?

Reply:
renamed it